src/main/java/it/gov/pagopa/pu/fileshare/util/FileUtils.java

```diff
@@ -14,27 +12,18 @@ public class FileUtils {
   private FileUtils() {
   }
 
-  public static String calculateFileHash(File file)
-    throws NoSuchAlgorithmException, IOException {
-    try (InputStream inputStream = new FileInputStream(file)) {
-      return calculateHash(inputStream);
-    }
-  }
-
-  public static String calculateFileHash(InputStream inputStream)
-    throws NoSuchAlgorithmException, IOException {
-    return calculateHash(inputStream);
+  public static String calculateBase64FileHash(byte[] hash) {
+    return Base64.getEncoder().encodeToString(hash);
   }
```

Comment:
This method is more generic than its name:
```suggestion
  public static String encodeBase64(byte[] value) {
    return Base64.getEncoder().encodeToString(value);
  }
```